src/Command/AssertBackwardsCompatible.php

```diff
@@ -149,7 +151,11 @@ public function execute(InputInterface $input, OutputInterface $output) : int
             ? $this->parseRevisionFromInput($input, $sourceRepo)
             : $this->determineFromRevisionFromRepository($sourceRepo, $stdErr);
 
-        $toRevision = $this->parseRevision->fromStringForRepository($input->getOption('to'), $sourceRepo);
+        $to = $input->getOption('to');
+
+        assert(is_string($to));
```

Comment:
Can we rely on the end user's environment having `assert` enabled, or should we use `beberlei/assert`?

Reply:
I went with `assert()` for psalm compatibility here: the chance of a type mismatch (due to misconfiguration) are extremely low, so a simple `assert()` would work here.

This is a regression caused by symfony widening the type of `getOption` to `string|int|float|null|array` in newer releases.

Reply:
note that Symfony only fixed their phpdoc to account for the different cases depending on the config of the option.

Reply:
Well aware: this is just documentation getting closer to reality :+1: